src/test/java/ai/chalk/arrow/TestUnmarshaller.java

```diff
@@ -13,18 +15,31 @@
 import org.apache.arrow.vector.table.Table;
 import org.apache.arrow.vector.types.TimeUnit;
 import org.apache.arrow.vector.types.pojo.FieldType;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
 import org.junit.jupiter.api.Test;
 
 import java.time.*;
 import java.util.*;
 
 
 public class TestUnmarshaller {
+    private static RootAllocator allocator;
+    @BeforeAll
```

Comment:
consider doing before/after each, would let you know if a singular test has a leak vs. all tests

Reply:
addressed